pyproject.toml

```diff
@@ -35,6 +35,12 @@ dependencies = [
 
     # Multimodal
     "Pillow>=9.4.0",
+
+    # Triton:
+    "triton>=2.3.1 ; platform_system != 'Windows'",
```

Comment:
I don't believe we'll want these as a primary dependency as we try to keep this list as lean as possible for users. I would move these to the optional dependencies for dev users and we'll include instructions for the extra dependencies in the loss doc string.